packages/eas-cli/src/credentials/ios/appstore/authenticate.ts

```diff
@@ -144,6 +152,33 @@ async function loginWithUserCredentialsAsync({
 }
 
 export async function authenticateAsync(options: Options = {}): Promise<AuthCtx> {
+  if (options.mode === AuthenticationMode.API_KEY) {
+    return await authenticateWithApiKeyAsync(options);
+  }
+  return await authenticateAsUserAsync(options);
```

Comment:
```suggestion
  } else {
    return await authenticateAsUserAsync(options);
  }
```
It's easier to think about the branches in this case.